bestbook.egg-info/PKG-INFO

```diff
@@ -0,0 +1,98 @@
+Metadata-Version: 1.0
+Name: bestbook
+Version: 0.0.1
+Summary: Rigorous Book Recommendations
+Home-page: UNKNOWN
+Author: root
+Author-email: UNKNOWN
+License: LICENSE
+Description: # TheBestBookOn.com
+        
+        ![The Best Book On logo](https://thebestbookon.com/static/imgs/best-book-on.png)
+        
+        ## Philosophy
+        
+        Book preference is a matter of taste, however, many people have similar tastes. One book may be perceived as better than another book in several ways, and in a combinations of these 10 ways (shown below). While we may be able to say one book is better than other if it is outperformed unanimously in every category, it likely makes more sense to compare books under the context of a specific criteria, for instance, its reading level or accuracy. Learn more about this direction at [Open Book Genome Project](https://bookgenomeproject.org).
+        
+        ## Installation
+        
+        ```bash
+        virtualenv venv
```

Comment:
```suggestion
       python3 -m venv venv
```

Reply:
This is coming from the README, which should probably be updated once we have an MVP.